docker/setup-nginx

```diff
@@ -43,5 +43,7 @@ with open(conf_template, "r") as f:
 
     config = re.sub(r"{{(.+)}}", r"{{ .Env.\g<1> }}", config)
 
+    config = re.sub(r"^}\s\Z", "\n    include /etc/nginx/conf.d/iml-timer.extras;\n}", config, 0, re.MULTILINE)
```

Comment:
I wonder if nginx's config says something like `include /etc/nginx/conf.d/*`

Reply:
One thing I could do is create a folder like this:
```
/etc/nginx/conf.d/extras
```
and put in any extra location files in that directory. With that said I don't think it's necessary at the moment.